packages/rum-core/src/error-logging/error-logging.js

```diff
@@ -110,10 +110,32 @@ class ErrorLogging {
     }
   }
 
-  registerGlobalEventListener() {
+  registerListeners() {
     window.addEventListener('error', errorEvent =>
       this.logErrorEvent(errorEvent)
     )
+    window.addEventListener('unhandledrejection', promiseRejectionEvent =>
+      this.logPromiseEvent(promiseRejectionEvent)
+    )
+  }
+
+  logPromiseEvent(promiseRejectionEvent) {
+    const prefix = 'Unhandled promise rejection: '
+    const { reason } = promiseRejectionEvent
+
+    if (reason == null) {
+      this.logError(prefix + '<no reason specified>')
```

Comment:
Do other agents have a similar message here? Maybe it makes sense to use the same format for `no reason specified`

Reply:
Checked Node.js already, there is no message. we can agree on it later if it needed.